compiler/src/dmd/mars.d

```diff
@@ -40,6 +40,7 @@ import dmd.dsymbolsem;
 import dmd.dtemplate;
 import dmd.dtoh;
 import dmd.errors;
+import dmd.errorsink;
```

Comment:
Useless import

Reply:
Yes, that's leftover from a previous incarnation of this PR.